spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/handle/DragHandle.kt

```diff
@@ -19,36 +19,56 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package com.adevinta.spark.components.bottomsheet
+package com.adevinta.spark.components.bottomsheet.handle
 
-import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
 import com.adevinta.spark.PreviewTheme
+import com.adevinta.spark.R
 import com.adevinta.spark.SparkTheme
+import com.adevinta.spark.components.bottomsheet.BottomSheetDefaults.DragHandleHeight
+import com.adevinta.spark.components.bottomsheet.BottomSheetDefaults.DragHandleTopPadding
+import com.adevinta.spark.components.bottomsheet.BottomSheetDefaults.DragHandleWidth
 import com.adevinta.spark.components.surface.Surface
 import com.adevinta.spark.tools.modifiers.sparkUsageOverlay
 import com.adevinta.spark.tools.preview.ThemeProvider
 import com.adevinta.spark.tools.preview.ThemeVariant
 
+/**
+ * The optional visual marker placed on top of a bottom sheet to indicate it may be dragged.
+ */
 @Composable
-public fun DragHandle(modifier: Modifier = Modifier) {
-    Box(
+public fun DragHandle(
+    modifier: Modifier = Modifier,
+    color: Color = SparkTheme.colors.outline.copy(alpha = SparkTheme.colors.dim1),
```

Comment:
```suggestion
    color: Color = SparkTheme.colors.outline.dim1,
```
We need to add the import also